__tests__/SharedValue.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { useSharedValue } from '../src';
+
+const ViewTestComponent = () => {
+  const sv = useSharedValue(1);
+  return <View testID="view">{sv.value}</View>;
+};
+
+function TestComponentWithSV() {
+  const sv = useSharedValue(1);
+
+  return (
+    <View>
+      <View testID="view">{sv.value}</View>
+      <Button
+        testID="button"
+        title="run"
+        onPress={() => {
+          sv.value = sv.value + 1;
+        }}
+      />
+    </View>
+  );
+}
+
+describe('SharedValue basic render', () => {
+  test('contains default value', () => {
+    const { getByTestId } = render(<ViewTestComponent />);
+    const view = getByTestId('view');
+    expect(view.children[0]).toBe('1');
```

Comment:
Won't `firstChild` work here?

Reply:
I didn't find an easier way to get the first child of element :/